### ISSUE TITLE
Update Matter.js?

### ISSUE THREAD

**Issue by goldfire** [photonstorm/phaser#3929]
Phaser 3 is currently using Matter.js v0.13.1, which is roughly a year old. The latest release is 0.14.2, which was released in June of this year. Is there any reason not to update this in Phaser?

**Comment by hexus** [comment]
I'd like to see it updated too. I'd also prefer Matter to be brought in via NPM, and simply aliased to the Phaser namespace.

**Comment by goldfire** [comment]
Yes, I definitely agree with the point about npm. I'm still not sure why it was done that way considering there are some other external libraries in the package.json.

**Comment by goldfire** [comment]
@photonstorm Thanks for the clarification, I wasn't sure if that was the case or not.

**Comment by vicksonzero** [comment]
may i know if the new matter.js integration will include `Body.setParts()` ?

**Comment by vicksonzero** [comment]
just to clarify: including `Body.setParts()` means it will attach `Phaser.Physics.Matter.Sprite` to another `Phaser.Physics.Matter.Sprite`, making them move together

**Comment by photonstorm** [comment]
Matter has now been updated to 0.14.2.
